### PR TITLE
Add daily runner

### PR DESCRIPTION
**Pull request by mgraber**

### CODE REVIEW

File: .github/workflows/daily.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    env:
+      RDP_S3_ENDPOINT: https://s3.amazonaws.com 
+      RDP_ACCESS_KEY_ID: ${{ secrets.RDP_ACCESS_KEY_ID }}
+      RDP_SECRET_ACCESS_KEY: ${{ secrets.RDP_SECRET_ACCESS_KEY }}
+      SG_S3_ENDPOINT: https://s3.wasabisys.com
+      SG_ACCESS_KEY_ID: ${{ secrets.SG_ACCESS_KEY_ID }}
+      SG_SECRET_ACCESS_KEY: ${{ secrets.SG_SECRET_ACCESS_KEY }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: install dependencies ...
```
Comment (SPTKL) [inline .github/workflows/daily.yml:21]: we just need to do `pip3 install -r requirements.txt` here

File: .github/workflows/daily.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        run: |
+          curl -O https://dl.min.io/client/mc/release/linux-amd64/mc
+          chmod +x mc
+          sudo mv ./mc /usr/bin
+          mc config host add sg $SG_S3_ENDPOINT $SG_ACCESS_KEY_ID $SG_SECRET_ACCESS_KEY --api S3v4
+          mc config host add rdp $RDP_S3_ENDPOINT $RDP_ACCESS_KEY_ID $RDP_SECRET_ACCESS_KEY --api S3v4
+
+      - name: Update daily_borough_poivisits_by_sector ...
+        working-directory: recipes
```
Comment (SPTKL) [inline .github/workflows/daily.yml:30]: we need to remove working directory for all, the file `./sg` is in root

File: .github/workflows/daily.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+name: Update daily tables
+
+on:
+  repository_dispatch:
+    types: [daily]
+  schedule:
+    - cron: 0 0 * * *
```
Comment (SPTKL) [inline .github/workflows/daily.yml:7]: should we run this slightly after we run `sync`? like 1 hour after?

**Review by SPTKL — CHANGES_REQUESTED** [review]

**Comment by mgraber** [comment]
I am currently running the historical quarters. We need to switch to the current quarter method in the py scripts before merging.

Maybe we can keep historical quarter version in another branch.

**Review by SPTKL — APPROVED** [review]

**Pull request merged**